windows/rngallery.sln

```diff
@@ -283,6 +283,18 @@ Global
 		{3E3931F2-4735-4417-8CB0-33668A7314D6}.Release|x64.Build.0 = Release|x64
 		{3E3931F2-4735-4417-8CB0-33668A7314D6}.Release|x86.ActiveCfg = Release|Win32
 		{3E3931F2-4735-4417-8CB0-33668A7314D6}.Release|x86.Build.0 = Release|Win32
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Debug|ARM64.Build.0 = Debug|ARM64
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Debug|x64.ActiveCfg = Debug|x64
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Debug|x64.Build.0 = Debug|x64
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Debug|x86.ActiveCfg = Debug|Win32
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Debug|x86.Build.0 = Debug|Win32
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Release|ARM64.ActiveCfg = Release|ARM64
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Release|ARM64.Build.0 = Release|ARM64
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Release|x64.ActiveCfg = Release|x64
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Release|x64.Build.0 = Release|x64
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Release|x86.ActiveCfg = Release|Win32
+		{193DF456-3F97-4CC1-A7D4-809857CC064E}.Release|x86.Build.0 = Release|Win32
```

Comment:
Not sure I understand why this part is here
```suggestion
```
The guid `193DF456-3F97-4CC1-A7D4-809857CC064E` points to the linear gradient. And appears to be in duplicate. Safe to remove?